incubator/postfix/templates/deployment.yaml

```diff
@@ -10,27 +10,66 @@ spec:
     metadata:
       labels:
         app: {{ template "fullname" . }}
+      annotations:
+        checksum/secret: {{ include (print $.Chart.Name "/templates/secret.yaml") . | sha256sum }}
     spec:
       containers:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         env:
+{{- range $name, $value := .Values.env }}
+{{- if not (empty $value) }}
+        - name: {{ $name | quote }}
+          value: {{ $value | quote }}
+{{- end }}
+{{- end }}
         - name: POSTMASTER_USER
+          value: "postmaster"
```

Comment:
Because docker image suggest only 1 user ( based on pam auth) it is - postmaster.
We do not have code that will create any additional users on startup.

Only change password  for postmaster.

So I want to reduce flexibility that only make confuse users